pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -44,7 +44,9 @@ std::unordered_set<std::string> GetTorchMlirWhiteList() {
       "aten::mul",      "aten::eq",          "aten::lt",
       "aten::ne",       "aten::__and__",     "aten::floor_divide",
       "prim::Constant", "aten::expand_as",   "aten::repeat",
-      "aten::expand",   "aten::sum",         "aten::mean"};
+      "aten::expand",   "aten::sum",         "aten::mean",
```

Comment:
Should we sort this list by name? and maybe one line one op is easier to maintain (need to clang-format off to protect)?

Reply:
I will resort the list in another PR.